core/services/ocr2/plugins/ocr2keeper/evm/log_filters.go

```diff
@@ -0,0 +1,102 @@
+package evm
+
+import (
+	"bytes"
+	"math/big"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/pkg/errors"
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/logpoller"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/generated/i_keeper_registry_master_wrapper_2_1"
+)
+
+const (
+	// logRetention is the amount of time to retain logs for.
+	// 5 minutes is the desired retention time for logs, but we add an extra 10 minutes buffer.
+	// TODO: TBD make this configurable or based on block time.
+	logRetention = (time.Minute * 5) + (time.Minute * 10)
+)
+
+// LogTriggerConfig is an alias for log trigger config.
+type LogTriggerConfig i_keeper_registry_master_wrapper_2_1.KeeperRegistryBase21LogTriggerConfig
```

Comment:
I think this is a new type definition rather than an alias, if we aren't decorating this type with extra functionality, should this become type LogTriggerConfig = ... ?